apps/portlet-documents/src/main/webapp/vue-app/attachment/components/AttachmentsDrawer.vue

```diff
@@ -191,6 +191,9 @@ export default {
     },
     attachmentPrivacyLabel() {
       return `${this.attachedFromOtherDrivesLabel} ${this.attachmentsWillBeDisplayedForLabel}`;
+    },
+    filesUploadedSuccessLabel() {
+      return this.entityType && this.entityId ? this.$t('attachments.upload.success') : this.$t('documents.upload.success');
```

Comment:
just to let you know, you can use this expression instead (the JS minifier will do it for you anyway)
```suggestion
      return this.entityType && this.entityId && this.$t('attachments.upload.success') || this.$t('documents.upload.success');
```